syntax = "proto3";

//java options
option java_multiple_files = false;
option java_package = "com.venkatasudha.entities";
option java_generate_equals_and_hash = true;

package entities;

message Users {
  int32 id = 1;
  string userName = 2;
  optional string firstName = 3;
  optional string lastName = 4;
}
